spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -138,7 +138,7 @@ public String deploy(AppDeploymentRequest request) {
 					instance.stdout.deleteOnExit();
 					instance.stderr.deleteOnExit();
 				}
-				logger.info("deploying app {} instance {}\n   Logs will be in {}", deploymentId, i, workDir);
+				logger.info("Deploying app with deploymentId {} instance {}.\n   Logs will be in {}", deploymentId, i, workDir);
```

Comment:
We can make the similar change [here](https://github.com/spring-cloud/spring-cloud-deployer-local/pull/43/files#diff-45e9460cf3b42d1da7edb5b11116483dR97) as well.